addon/utils/has-valid-field-options.js

```diff
@@ -0,0 +1,30 @@
+export function hasValidFieldOptions(
+  fieldModel,
+  requiredProperties = ['conceptScheme']
```

Comment:
Should we remove this default? So it can also be used for ... if needed (not the case atm i think)
* `ext:alertField` => requires some properties (didn't check which)
* `ext:headerField` => not required to add them
* `ext:inputField` => not required to add them
* `ext:caseNumberField` => not required to add them (but will result in weird `null1234` output in dummy app)

Reply:
Actually should remove it as the name of the method does not state that this will check for the conceptScheme key as default. Leaving it atm till I have a response

Reply:
Removing it makes sense, yea.